### PR TITLE
Fix crash when clicking on xsheet cell

### PR DESCRIPTION
**Pull request by shun-iwasawa**
This PR is for the issue #425

### CODE REVIEW

**Comment by shun-iwasawa** [comment]
Jenkins

**Comment by kogaki** [comment]
Jenkins, again

File: toonz/sources/toonz/xshcellviewer.cpp [hunk 1]
```diff
@@ -797,6 +797,9 @@ void CellArea::drawCells(QPainter &p, const QRect toBeUpdated)
 	}
 
 	// smart tab
+	// initialize the rectangles first
+	m_levelExtenderRect = QRect();
+	m_upperLevelExtenderRect = QRect();
```
Comment (kogaki) [inline toonz/sources/toonz/xshcellviewer.cpp]: I feel this should be done in constructor.

Comment (kogaki) [inline toonz/sources/toonz/xshcellviewer.cpp]: OK. I now understand this is not for initialization, but for clear.

**Comment by kogaki** [comment]
LGTM. It's surprising to be able to find such a bug. @shun-iwasawa

**Pull request merged**